compute_endpoint/globus_compute_endpoint/endpoint/endpoint_manager.py

```diff
@@ -473,32 +531,47 @@ def _event_loop(self):
                 f"\n  Globus effective identity: {effective_identity}"
                 f"\n  Globus identity set: {identity_set}"
             )
-            try:
-                local_username = self.identity_mapper.map_identity(identity_set)
-                if not local_username:
-                    raise LookupError()
-            except LookupError as e:
-                log.error(
-                    "Identity failed to map to a local user name."
-                    f"  ({e.__class__.__name__}) {e}{identity_for_log}"
-                )
-                continue
-            except Exception as e:
-                msg = "Unhandled error attempting to map user."
-                log.debug(f"{msg}{identity_for_log}", exc_info=e)
-                log.error(f"{msg}  ({e.__class__.__name__}) {e}{identity_for_log}")
-                continue
 
-            try:
-                local_user_rec = pwd.getpwnam(local_username)
+            local_user_rec = None
+            local_username = None
+            if not self.identity_mapper or parent_identities:
+                # we are not a privileged user, so allow _only_ the identity of the
+                # parent process auth'd to run tasks
 
-            except Exception as e:
-                log.error(
-                    f"({type(e).__name__}) {e}\n"
-                    "  Identity mapped to a local user name, but local user does not"
-                    f" exist.\n  Local user name: {local_username}{identity_for_log}"
-                )
-                continue
+                if not parent_identities.intersection(identity_set):
```

Comment:
Does the user get any feedback besides this log line that the request failed?

Reply:
That's good question.  Per our phone conversation, they do now.